src/app/collective-rewards/user/context/BuilderContext.tsx

```diff
@@ -77,6 +78,7 @@ export const BuilderContextProvider: FC<BuilderProviderProps> = ({ children }) =
           proposalName,
           proposalDescription,
           joiningDate,
+          gauge: builder.gauge,
```

Comment:
I would destruct the builder object in line 62. 

` const { status, address, gauge } = builder`

And just pass the property:
` gauge`